PyInstaller/building/api.py

```diff
@@ -636,13 +637,21 @@ def assemble(self):
             exe = tmpnm
         exe = checkCache(exe, strip=self.strip, upx=self.upx)
         self.copy(exe, outf)
-        if self.append_pkg:
-            logger.info("Appending archive to EXE %s", self.name)
-            self.copy(self.pkg.name, outf)
+        if subprocess.check_call(['objcopy','--help']) != 0:
+            if self.append_pkg:
+                logger.info("Appending archive to EXE %s", self.name)
+                self.copy(self.pkg.name, outf)
+            else:
```

Comment:
@htgoebel Do we still use the `append_pkg`? I think this feature is already dropped?